### PR TITLE
Fix set HELPURLs correctly

### LINKED ISSUE #1366: Logical or block has help menu item disabled
In testing #1358 I noticed that if one right-clicks on the logical or block, the help menu item is disabled. The problem is that in logical_or, the HELPURLS and TOOLTIP functions are not called prior to accessing the operator, and since "AND" and "OR" are not members of the Function type, `undefined` is returned. Lines 228 and 233 of logic.js need to be changed to call the method prior to the lookup (cf. lines 169 and 174).

### PR DESCRIPTION
**Pull request by SusanRatiLane**
Set HELPURL correctly for several multi-operation math and logic blocks.

Added HELPURL definition for min and max.

Fixes #1366

### CODE REVIEW

**Comment by ewpatton** [comment]
@SusanRatiLane File it as a separate issue and we can fix it after the other PRs are merged. I expect it isn't the only block and documentation have diverged. It might make sense to do an audit of all of the documentation to understand what's out of date with master.

**Comment by SusanRatiLane** [comment]
Ooooh. I tested the convert radians/degrees block. I didn't realize there was another convert block.

**Comment by SusanRatiLane** [comment]
OK, convert_number should be working now.

File: appinventor/blocklyeditor/src/blocks/math.js [hunk 1]
```diff
@@ -1056,6 +1066,7 @@ Blockly.Blocks['math_convert_number'] = {
   helpUrl: function() {
       var mode = this.getFieldValue('OP');
       return Blockly.Blocks.math_convert_number.HELPURLS[mode];
+//      return Blockly.Blocks.math_convert_number.HELPURLS()[mode];
```
Comment (ewpatton) [inline appinventor/blocklyeditor/src/blocks/math.js]: If this line isn't being used, we should just take it out.

Comment (SusanRatiLane) [reply appinventor/blocklyeditor/src/blocks/math.js]: OK, this is done.

**Pull request merged**